packages/flutter/lib/src/material/radio_list_tile.dart

```diff
@@ -228,7 +228,13 @@ class RadioListTile<T> extends StatelessWidget {
           isThreeLine: isThreeLine,
           dense: dense,
           enabled: onChanged != null,
-          onTap: onChanged != null ? () { onChanged(value); } : null,
+          onTap: onChanged != null ?
```

Comment:
I also couldn't decide between this and:
```
onTap: onChanged != null  && !checked ? () { onChanged(value); } : null,
```

I'm leaning towards the current implementation a little, since it's less cluttery

Reply:
I think this one-liner is a little easier to read